integreat_cms/cms/views/authentication/totp_login_view.py

```diff
@@ -51,6 +51,29 @@ def dispatch(self, request, *args, **kwargs):
         # Now process dispatch as it otherwise normally would
         return super().dispatch(request, *args, **kwargs)
 
+    def get(self, request, *args, **kwargs):
+        r"""
+        Renders the login form
+
+        :param request: The current request
+        :type request: ~django.http.HttpRequest
+
+        :param \*args: The supplied arguments
+        :type \*args: list
+
+        :param \**kwargs: The supplied kwargs
+        :type \**kwargs: dict
+
+        :return: Rendered login form
+        :rtype: ~django.http.HttpResponse
+        """
+
+        return render(
+            request,
+            self.template_name,
+            {"can_use_webauthn": self.user.mfa_keys.exists()},
```

Comment:
If you just want to add a template context, it's not neceessary to overwrite `get()` - just use [`get_context_data()`](https://docs.djangoproject.com/en/4.2/topics/class-based-views/generic-display/#adding-extra-context) instead.
This also has the advantage that you can use
```
        return render(
            request,
            self.template_name,
            self.get_context_data(**kwargs),
        )
```
in the post view as well, and have the same context variable even if the form validation fails and the post request is rendered again without a call to `get()`.